api/routes/course.js

```diff
@@ -1,5 +1,16 @@
 const express = require("express");
+const Course = require("../models/Course");
 
 const router = express.Router();
 
+
+router.get('/', async (req, res) => {
```

Comment:
the API for getting all courses will be more complicated. We want to request all courses that the user has. I have done it, but not merged yet